src/components/pastAnalysis/PastAnalysisTable.tsx

```diff
@@ -28,23 +26,74 @@ interface tableRowsChild {
   cells: { [title: string]: ReactNode }[]
 }
 
-interface TableStates {
-  page: number,
-  maxFeedId: number | undefined,
-  lastOffset: number,
-  lastPage: number,
-  storedPages: StudyInstanceWithSeries[][],
-  processingPluginIds: number[]
+type TableState = {
+  page: number, // Current table page number
+  maxFeedId: number | undefined, // ID of the latest Feed on Swift as of when PastAnalysisTable first mounted OR was last reset
+  lastOffset: number, // Page offset value for where to begin fetching the next unseen page
+  lastPage: number, // Table page number of the very last page (-1 means last page has not yet been seen)
+  storedPages: StudyInstanceWithSeries[][], // Stores pages that have been seen in an array of pages
+  processingPluginIds: number[] // Stores plugin ids associated with images that are currently processing, used for selective polling
+}
+
+const initialTableState: TableState = {
+  page: 0,
+  maxFeedId: -1,
+  lastOffset: 0,
+  lastPage: -1,
+  storedPages: [],
+  processingPluginIds: []
+}
+
+enum TableReducerActions {
+  updateMaxFeedId = "UPDATE_MAX_FEED_ID",
+  addNewPage = "ADD_NEW_PAGE",
+  incrementPage = "INCREMENT_PAGE",
+  decrementPage = "DECREMENT_PAGE"
```

Comment:
I would highly suggest using capitalized words separated by underscores for action types since it is the widely used practice adopted by the Redux, Context and other state management communities.